package.json

```diff
@@ -10,7 +10,8 @@
     "preview": "npm run preview:liveion",
     "preview:liveion": "vite preview -c web/liveion/vite.config.ts",
     "preview:liveman": "vite preview -c web/liveman/vite.config.ts",
-    "build": "npm run build:liveion && npm run build:liveman",
+    "lint": "eslint && tsc --noEmit",
```

Comment:
In my computer:

```bash
npm run lint
```

Why show error?

```
...
  9:440   warning  Missing semicolon                                                          @stylistic/js/semi
  9:498   error    'clearTimeout' is not defined                                              no-undef
  9:597   error    Strings must use singlequote                                               @stylistic/js/quotes
  9:647   error    Expected an assignment or function call and instead saw an expression      @typescript-eslint/no-unused-expressions
  9:676   error    Strings must use singlequote                                               @stylistic/js/quotes
  9:704   error    'fetch' is not defined                                                     no-undef
  9:735   error    Strings must use singlequote                                               @stylistic/js/quotes
  9:755   warning  Missing semicolon                                                          @stylistic/js/semi

✖ 35888 problems (10000 errors, 25888 warnings)
  7577 errors and 25888 warnings potentially fixable with the `--fix` option.
```

Reply:
It reports those errors on which file? I can't reproduce it locally, and CI seems to work fine.